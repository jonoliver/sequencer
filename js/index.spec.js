import React from 'react';
import { mount } from 'enzyme';
import Main from '.';
import renderer from 'react-test-renderer';

it('displays the message (snapshot)', () => {
  const tree = renderer
    .create(<Main />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

xit('displays the message (enzyme)', () => {
  const main = mount(<Main />);
  expect(main.find("p").text()).toBe("Hello World!")
});
