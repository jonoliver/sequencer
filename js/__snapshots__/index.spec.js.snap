// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the message (snapshot) 1`] = `
<div
  className="container"
  onMouseDown={[Function]}
  onMouseUp={[Function]}
>
  <div
    className="grid"
  >
    <div
      className="column"
    >
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
    </div>
    <div
      className="column"
    >
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
    </div>
    <div
      className="column"
    >
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
    </div>
    <div
      className="column"
    >
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell active"
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
      <div
        className="cell "
        onClick={[Function]}
        onMouseEnter={[Function]}
      />
    </div>
  </div>
</div>
`;
